global.json

```diff
@@ -28,6 +28,9 @@
   },
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "10.0.0-beta.24560.1",
-    "Microsoft.DotNet.Helix.Sdk": "10.0.0-beta.24560.1"
+    "Microsoft.DotNet.Helix.Sdk": "10.0.0-beta.24560.1",
+    "Microsoft.DotNet.SharedFramework.Sdk" : "10.0.0-beta.24560.1",
+    "Microsoft.Build.NoTargets": "3.7.0",
```

Comment:
Why these versions/when should we update them?

Reply:
These are the versions used by dotnet/runtime today. I believe they're the newest versions. We generally try to update them when new versions come out, but we don't have a regular cadence.